runner/models.py

```diff
@@ -1,9 +1,10 @@
 import uuid
 from django.db import models
-
+from django.db.models.fields import CharField
+from django.db.models.fields.files import ImageField
+from django.db.models.fields.related import ForeignKey
```

Comment:
Guessing those imports are gonna be used later @xbello?

Reply:
Fields are already imported along "db.models". I don't think we need to be ultra-specific here. For now we can leave it this way, it doesn't hurt.